recipes/libcf/build.sh

```diff
@@ -0,0 +1,15 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
```

Comment:
hum... In general I'm reluctant to picking up the user's flags... Does your script sets CFALGS in any special ways?